sdk/event/grpc_client.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
For all the gRPC files, and the plugin's main package, I don't think it's possible to actually use it without the ability to register an external plugin? Do say if I'm wrong, but I think maybe we should defer adding these bits until we enable external plugins to avoid confusion and dead/untested code.

Reply:
Oh, that makes sense, and will simplify this PR. We can go ahead and remove the gRPC stuff for now.